src/index.ts

```diff
@@ -92,7 +92,7 @@ export const rehydrateApplicationState = (
       const isObjectRegex = new RegExp('{|\\[');
       let raw = stateSlice;
 
-      if (isObjectRegex.test(stateSlice.charAt(0))) {
+      if (isObjectRegex.test(stateSlice.charAt(0)) || stateSlice === 'null') {
```

Comment:
I'd put your condition before regex test - just for optimisation :smiley:

Reply:
i'll look into it now

Reply:
did that update
@btroncone could you please merge this?